test/sql/catalog/function/attached_macro.test

```diff
@@ -0,0 +1,47 @@
+# name: test/sql/catalog/function/attached_macro.test
+# group: [function]
+
+require skip_reload
+
+require httpfs
+
+statement ok
+ATTACH 'https://blobs.duckdb.org/data/checksum-macro.duckdb' AS checksum_macro;
```

Comment:
I think this works the same (= was broken in the same way) also for local dbs, so I would probably remove the httpfs dependency